spinta/exceptions.py

```diff
@@ -1067,3 +1067,11 @@ class UnableToFindPrimaryKeysMultipleUniqueConstraints(UserError):
     Multiple `UniqueConstrains` were found (that do not match new model's primary keys):
     {unique_constraints}
     '''
+
+
+class NestedPropertyDefinedWithoutReferencePropertyError(UserError):
```

Comment:
Gal galima sugalvoti trumpesnį pavadinimą, pavyzdžiui:

```suggestion
class ParentNodeNotFound(UserError):
```

?

Reply:
Atlikau siūlomus pakeitimus: [423a8ee](https://github.com/atviriduomenys/spinta/pull/1181/commits/423a8eed1565515c153dd30dc65a09da9e528f99)